recipe/meta.yaml

```diff
@@ -56,6 +56,7 @@ test:
     - pytest
     - pytorch
     - torchvision
+    - openvino
```

Comment:
Even coremltools should be installed by  conda

https://github.com/conda-forge/coremltools-feedstock

If it is required for tests there are really a few options:

1. Skip the tests on conda-forge
2. Update the package

Reply:
@hmaarrfk thanks for the review! Good to know coremltools is available on Conda. I'll add it to the list of packages needed for tests.